### ISSUE TITLE
Incorrectly blocking fass.se.

### ISSUE THREAD

**Issue by jav** [MetaMask/eth-phishing-detect#3629]
Fass.se is a swedish medical information site.
It's wrongly blocked by eth-phising-detect.

**Comment by 409H** [comment]
Closing. Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/3667
